packages/admin/admin/src/messages.ts

```diff
@@ -54,4 +54,7 @@ export const messages = defineMessages({
     deleteItem: { id: "comet.generic.deleteItem", defaultMessage: "Delete Item" },
     empty: { id: "comet.generic.empty", defaultMessage: "Empty" },
     downloadAsExcel: { id: "comet.generic.downloadAsExcel", defaultMessage: "Download as Excel" },
+    impersonate: { id: "comet.generic.impersonate", defaultMessage: "Impersonate" },
+    startImpersonation: { id: "comet.generic.startImpersonation", defaultMessage: "Start Impersonation" },
+    stopImpersonation: { id: "comet.generic.stopImpersonation", defaultMessage: "Stop Impersonation" },
```

Comment:
Should this really be common messages, even if they're used in `@comet/cms-admin` only?

Reply:
Thank you! This totally makes sense to me now.

Updated here: 233610eafd3fbc407e0b5172f9b1945ea0bd9deb & 4768ee4d901182bf3450227911929b9d491e9028